pkg/cri/io/logger.go

```diff
@@ -143,7 +143,11 @@ func redirectLogs(path string, rc io.ReadCloser, w io.Writer, s StreamType, maxL
 			lineBuffer.Write(l)
 		}
 		lineBuffer.WriteByte(eol)
-		if _, err := lineBuffer.WriteTo(w); err != nil {
+		n, err := lineBuffer.WriteTo(w)
+		if err == nil {
+			outputEntries.Inc()
+			outputBytes.Inc(float64(n))
+		} else {
```

Comment:
nit: keep `n`, and `err` scoped to the if

```suggestion
		if n, err := lineBuffer.WriteTo(w); err == nil {
			outputEntries.Inc()
			outputBytes.Inc(float64(n))
		} else {
```

Reply:
Done